exporter/pulsarexporter/factory.go

```diff
@@ -24,7 +24,12 @@ import (
 )
 
 const (
-	typeStr             = "pulsar"
+	// The value of "type" key in configuration.
+	typeStr = "pulsar"
+
+	// The stability level of the exporter.
+	stability = component.StabilityLevelBeta
```

Comment:
Is it beta? Docs says it's in development. I would suggest updating in to Alpha in both places

Reply:
Done.